convert.py

```diff
@@ -90,12 +90,26 @@ def convert(args):
         for block in inference_program.blocks:
             for op in block.ops:
                 if op.type in ops.node_maker:
+                    op_attrs = dict([(name, op.attr(name))
+                                     for name in op.attr_names
+                                     ]) if op.attr_names is not None else None
+                    op_inputs = dict([(name, op.input(name))
+                                      for name in op.input_names])
+                    op_outputs = dict([(name, op.output(name))
+                                       for name in op.output_names])
+
+                    # Append some customized arguments of the node maker here
+                    if op.type == 'conv2d':
+                        kernel_shape = fluid.executor.fetch_var(
```

Comment:
Since this is all static, we could make `fetch_var` available inside the `ops` module, so we don't have to such a conditional here

Reply:
Agree. We should make `convert.py` more clean. So I change to pass operator and scope as arguments.